### ISSUE TITLE
Something went wrong in package update:  R 3.2.0 -> 3.2.1

### ISSUE THREAD

**Issue by VMJim** [talgalili/installr#34]
Running `installr` from the basic RGui on windows 8.1, I encountered the following error & warnings:

```
The file was downloaded succesfully into:
 C:\some\path\Local\Temp\Rtmp46FtSO/R-3.2.1-win.exe 

Running the installer now...

Installation status:  TRUE . Removing the file:
 C:\some\path\Local\Temp\Rtmp46FtSO/R-3.2.1-win.exe 
 (In the future, you may keep the file by setting keep_install_file=TRUE) 
Loading required package: tools
files ‘bin/R.exe’, ‘bin/Rscript.exe’ have the wrong MD5 checksums
No packages to copy.  Goodbye :) 
[1] TRUE
Warning messages:
1: In install.R(to_checkMD5sums = to_checkMD5sums, keep_install_file = keep_install_file,  :
  There was some problem with installing R.  Some files are not what they should be (e.g: check MD5 sums did not pass all the tests). 
  You can try installing R again (either manually or through install.R()), 
  and if the problem persists you can file a bug report by running:  bug.report(package = 'installr') 
2: running command 'C:/PROGRA~1/R/R-3.2.1/bin/Rscript.exe  -e " setInternet2(TRUE); options(repos=structure(c(CRAN='http://cran.rstudio.com/'))); update.packages(checkBuilt=TRUE, ask=F) "' had status 1
```
I have no clue what's going on here, but in any case it may be helpful feedback.

**Comment by talgalili** [comment]
HI @VMJim ,
Are you using installr 0.16.0 or earlier?

**Comment by VMJim** [comment]
Just checked: I'm using version 0.16.0.

**Comment by talgalili** [comment]
I think you might be seeing the package after it was updated, and that when running the updateR you used an older version. This is a known issue that was fixed in 0.16.0 (the issue didn't cause any problem, but still).

If this happens again in the next R version update, let me know :)

Best,
Tal
